RegistryMetadata.md

```diff
@@ -0,0 +1,26 @@
+# Registry Metadata
+
+## Introduction
+
+We have 3 data exchange mechanisms as described in the [Architecture Principles](README.md).  In as much as possible, we want to create the same metadata for each method, and expose that metadata through a common query device: The SmartAPI registry.   Below, these methods are abbreviated as S (for SmartAPI), T (for Translator Reasoner API), and G (for Knowledge Graph Exchange).  
+
+This document describes, broadly, the particular metadata that Translator data exchange methods must or should expose. The technical details of how this metadata is specified is located [here]() for SmartAPI-type interfaces, [here]() for Translator Reasoner APIs, and [here]() for Knowledge Graph Exchange files.  
+
+## Change Management
+
+Changes to this document must be made via pull requests.   Questions or discussion around a topic that is not easily related to a specific pull request occurs in github issues.
+
+## Metadata
```

Comment:
For those who were not at the KGE WG meeting last week, here is a [strawman for KG metadata](https://hackmd.io/WS-TmgYTQJaqtZycc9bHbw?view) proposed by @deepakunni3 that is relevant to this PR.

Reply:
I suppose that one point to be made here is whether or not such metadata will be embedded as a specialized subgraph in the knowledge graph itself, or  maintained as a separate metadata file alongside the KGE formated of the knowledge graph.